test/ts-scenario/steps/lib/listeners.ts

```diff
@@ -90,6 +98,7 @@ export async function createBlockListener(gatewayName: string, channelName: stri
 }
 
 export async function newFileCheckpointer(): Promise<Checkpointer> {
+	// eslint-disable-next-line @typescript-eslint/restrict-plus-operands
```

Comment:
Fix instead of disable rule if possible

Reply:
should it be os.tmpdir() instead?

Reply:
Yes, that looks like a bug, and it should have been using `os.tmpdir()`. Definitely worthwhile running the linting!

Reply:
It looks like the same bug is there in `fabric-network/test/testutils.ts` too